### ISSUE TITLE
data_to_json changes

### ISSUE THREAD

**Issue by jagthedrummer** [codeforokc/openbudgetokc#25]
@mascott would love to hear your thoughts on this idea.

In `sandbox/data_to_json.rb` we're asking people to identify things like 'account', 'fund', 'operating unit', etc...

I'm not sure that those terms will have any real inherent meaning to people. I'm wondering if we could do something like:

> Please select the number corresponding top level of categorization:
> Please select the number corresponding second level of categorization:
> Please select the number corresponding third level of categorization:
> Etc...

**Comment by mascott** [comment]
Yeah, that's absolutely the goal! That particular script was thrown together quickly when I was still trying to get a grasp of the data. 

I think it'd be great if the first question was "How many levels does your data contain?" and then spin through matching the header to the level.

We'll probably have graphs that need only a subset of the levels so determining which to use will be interesting.

**Comment by mascott** [comment]
Well, that was the goal...

Now the goal is to strip it down and make it fit our data exactly so it's easy to run. We can adjust it to be more flexible in the future.

**Comment by jagthedrummer** [comment]
We also need to move this script out of the sandbox and write some documentation around how to use it and where to place the source files, etc...
